pyproject.toml

```diff
@@ -11,6 +11,8 @@ keywords = ["fastapi", "oidc", "authentication", "authorization", "saml"]
 [tool.poetry.dependencies]
 python = "^3.6"
 fastapi = "^0.63.0"
+itsdangerous = "^2.0.1"
+uvicorn = "^0.14.0"
```

Comment:
uvicorn should not be a dependency in my opinion. It's not used except for the examples. If you want to keep it please add a extra for demo purposes.

Reply:
Deleted